File: solutions/0058. Length of Last Word/solution.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        count = 0
        for char in reversed(s):
            if char == ' ':
                if count > 0:
                    return count
            else:
                count += 1
        return count

File: solutions/0058. Length of Last Word/solution.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int count = 0;
        for (std::string::reverse_iterator rit=s.rbegin(); rit!=s.rend(); ++rit){
            if(*rit == ' '){
                if(count > 0){
                    return count;
                }
            }
            else{
                count++;
            }
        }
        return count;
    }
};
